elements/rh-table/rh-table.ts

```diff
@@ -0,0 +1,168 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+
+import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
+
+import styles from './rh-table.css';
+import { RequestSortEvent, RhSortButton } from './rh-sort-button.js';
+export * from './rh-sort-button.js';
+
+/**
+ * Displays tabular information in an easy and digestable manner.
+ *
+ * @summary
+ *
+ * @slot            - an HTML table
+ * @slot    summary - a brief description of the data
```

Comment:
Add a @summary

As well @csspart and @cssprop usage here as well

Reply:
Waiting for @summary from Corey

Reply:
Added ✔️